cli/src/pcluster/models/cluster_config.py

```diff
@@ -885,7 +884,10 @@ def _register_storage_validators(self):
                         os=self.image.os,
                     )
                 if isinstance(storage, SharedEbs):
-                    storage_count["ebs"] += 1
+                    if storage.raid:
+                        storage_count["raid"] += 1
```

Comment:
if changing the previous loop you can use here:
`storage_count[SharedStorageType.RAID]`.

This helps avoiding typos.